project-reports/2023/2023-Q3-Hyperledger-Indy.md

```diff
@@ -0,0 +1,154 @@
+[Indy HIPE]: https://github.com/hyperledger/indy-hipe
+
+## Project Health
+
+The project had less activity this quarter than previously, with fewer updates
+from (most) of the same organizations. The community had an Indy Ecosystem
+Summit on June 29, 2023, with over 50 participants talking about what they are
+doing with Indy. A follow up meeting to be held on September 7, 2023 is planned
+to convert the interest expressed in the first meeting into action, ideally
+based on the [Indy Roadmap].
+
+[Indy Roadmap]: https://hackmd.io/GeRP00i0Sj-7z4zXn2MB5g?view
+
+Per the [Indy Quarterly Activity Dashboard], there were 97 commits
+(the same as last quarter) from 24 contributors (almost double last quarter).
+
+[Indy Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Findy/dashboard;subTab=technical?time=%7B%22from%22:%222023-04-01T08:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222023-06-30T07:00:00.000Z%22%7D
+
+> NOTE: As of 2023.08.02 only some of the Indy repositories are visible on
+LFX Insights. The additional repositories are being added.
+
+## Questions/Issues for the TOC
+
+None
+
+### Issues from previous reports
+
+#### **Diversity of Contributor Community**
+
+See the updated type of information in the appropriate section of this report.
+
+## Releases
+
+- indy-vdr -  - v0.4.0-dev.13 to v0.4.0-dev.16
+- indy-node-container - v1.2.5
+- indy-shared-rs - v0.3.3
+
+## Overall Activity in the Past Quarter
+
+Activity was focused on the so-called Shared Components ([indy-vdr],
+[indy-shared-rs] and [aries-askar]) and the transition away from the Indy SDK. The
+maintainer community has discussed the formal deprecation of the Indy SDK and
+the Aries maintainers have been busy making the Shared Components the default in
+Aries, in preparation for the deprecation announcement.  
+
+The archiving of the Hyperledger Ursa project triggered a refresh of the underlying
+CL-Signatures code base (now [anoncreds-clsignatures-rs]), and a number of
+improvements along with a couple of bug fixes. The release as part of
+indy-shared-rs was the first upgrade to CL Signatures in several years, and
+included a speed up of 50% in creating a revocation registry, and 25% in the
+other revocation operations (issuance and revocation).
+
+[anoncreds-clsignatures-rs]: https://github.com/hyperledger/anoncreds-clsignatures-rs
+
+Little changed with the Indy Node implementation beyond further deployments of
+the software and confirmation of its stability. However, no migrations of
+existing networks to the Ubuntu 20.04 release have been performed. Testing,
+experimenting and documenting that process continues.
+
+[indy-vdr]: https://github.com/hyperledger/indy-vdr
+[aries-askar]: https://github.com/hyperledger/aries-askar
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+
+## Current Plans
+
+The second Indy Ecosystem Summit in September will be a key event as we look to
```

Comment:
I like this idea of ecosystem summit. It will be great to share this concept with other projects too.